go.mod

```diff
@@ -8,6 +8,12 @@ require (
 	github.com/imdario/mergo v0.3.8
 	github.com/olekukonko/tablewriter v0.0.2
 	github.com/spf13/cobra v0.0.5
+	github.com/spf13/pflag v1.0.5 // indirect
```

Comment:
The `tools-update` target is installing some test dependencies that end up introducing all these indirect references..  If I remember correctly, these caused us some merge issues early on.  

If I delete all the refs in `go.mod` after running `make` and then run a `go mod tidy`, they go away.  Is there a good way to automatically do this after the test run?